js/payment-methods/litecoin.js

```diff
@@ -0,0 +1,219 @@
+var app = app || {};
+
+app.paymentMethods = app.paymentMethods || {};
+
+app.paymentMethods.litecoin = (function() {
+
+	'use strict';
+
+	return app.abstracts.PaymentMethod.extend({
+
+		label: 'Litecoin',
+		code: 'LTC',
+
+		settings: [
+			{
+				name: 'xpub',
+				label: 'Master Public Key',
+				type: 'text',
+				required: true,
+				validate: function(value) {
+					try {
+						var node = app.paymentMethods.litecoin.getHDNodeInstance(value);
+					} catch (error) {
+						console.log(error);
+					}
+					if (!node) {
+						throw new Error('Invalid master public key');
+					}
+				}
+			},
+			{
+				name: 'scheme',
+				label: 'Address Path',
+				type: 'text',
+				default: 'm/0/n',
+				required: true
+			}
+		],
+
+		generatePaymentRequest: function(amount, cb) {
+
+			this.getPaymentAddress(function(error, address) {
+
+				if (error) {
+					return cb(error);
+				}
+
+				var paymentRequest = 'litecoin:' + address + '?amount=' + amount;
+				cb(null, paymentRequest, address);
+			});
+		},
+
+		checkPaymentReceived: function(paymentRequest, cb) {
+
+			_.defer(_.bind(function() {
+
+				var matches = paymentRequest.match(/litecoin:([a-zA-Z0-9]+)\?amount=([0-9\.]+)/);
+
+				if (!matches) {
+					return cb(new Error('Invalid payment request'));
+				}
+
+				var address = matches[1];
+				var amount = matches[2];
+
+				/*
+					For API details:
+					https://chain.so/api#get-balance
+				*/
+				var uri = 'https://chain.so/api/v2/get_address_balance';
+				// Network (e.g LTC or LTCTEST):
+				uri += '/LTC';
+				// Address:
+				uri += '/' + encodeURIComponent(address);
+				// Minimum number of confirmations:
+				uri += '/0';
+
+				$.get(uri).then(function(result) {
+
+					try {
+						var amountReceived = new BigNumber('0');
+						amountReceived.plus(result.data.confirmed_balance);
+						amountReceived.plus(result.data.unconfirmed_balance);
+						// Convert to "whole" LTC.
+						amountReceived = amountReceived.dividedBy('100000000');
```

Comment:
at least the response I get from [LTC blockexplorer](https://chain.so) contains:
```javascript
 "unconfirmed_balance" : "0.00216438" 
```
therefore it is not needed to divide it